experimental/cuda2/cuda_device.c

```diff
@@ -194,6 +199,18 @@ static iree_status_t iree_hal_cuda2_device_create_internal(
         host_allocator, &device->device_allocator);
   }
 
+  if (iree_status_is_ok(status) &&
+      params->command_buffer_mode == IREE_HAL_CUDA_COMMAND_BUFFER_MODE_STREAM) {
+    status = iree_hal_cuda2_stream_command_buffer_create(
```

Comment:
I can't figure out what this is used for, but it shouldn't exist - we can't reuse command buffers like this or have global state - multiple pieces of code may be using the same device and submitting work that overlaps - just make new command buffers as you need them

Reply:
Ah, right, good catch! Missed this part when porting over existing code.. Changed to do the allocation where it's used.